katas/content/multi_qubit_gates/index.md

```diff
@@ -230,11 +230,11 @@ $$=\begin{bmatrix} 1 & 0 & 0 & 0 \\ 0 & 1 & 0 & 0 \\ 0 & 0 & 0 & 1 \\ 0 & 0 & 1
 This representation can be used to carry out calculations in Dirac notation without ever switching back to matrix representation:
 
 $$
-CNOT|10\rangle =
-\big(|00\rangle\langle00| + |01\rangle\langle01| + |10\rangle\langle11| + |11\rangle\langle10|\big)|10\rangle =$$
-$$=|00\rangle\langle00|10\rangle + |01\rangle\langle01|10\rangle + |10\rangle\langle11|10\rangle + |11\rangle\langle10|10\rangle =$$
-$$=|00\rangle\big(\langle00|10\rangle\big) + |01\rangle\big(\langle01|10\rangle\big) + |10\rangle\big(\langle11|10\rangle\big) + |11\rangle\big(\langle10|10\rangle\big) =$$
-$$=|00\rangle(0) + |01\rangle(0) + |10\rangle(0) + |11\rangle(1) = |11\rangle$$
+CNOT\ket{10} =
+\big(\ket{00}\bra{00} + \ket{01}\bra{01} + \ket{10}\bra{11} + \ket{11}\bra{10}\big)\ket{10} =$$
+$$=\ket{00}\braket{00|10} + \ket{01}\braket{01|10} + \ket{10}\braket{11|10} + \ket{11}\braket{10|10} =$$
+$$=\ket{00}\big(\braket{00|10}\big) + \ket{01}\big(\braket{01|10}\big) + \ket{10}\big(\braket{11|10}\big) + \ket{11}\big(\braket{10|10}\big) =$$
+$$=\ket{00}(0) + \ket{01}(0) + \ket{10}(0) + \ket{11}(1) = \ket{11}$$
```

Comment:
Oh boy, this must have been such a pain to go through and fix up! Lots of tricky cases here. Nice job though.... all looking good so far!

Reply:
\braket replacement was one the more manual edits, actually :-) Showed up as unpaired \langle-s and needed another round of fixing